src/platform/packages/shared/kbn-esql-ast/src/definitions/types.ts

```diff
@@ -214,7 +213,6 @@ export interface FunctionDefinition {
   validate?: (fnDef: ESQLFunction) => ESQLMessage[];
   operator?: string;
   customParametersSnippet?: string;
-  // Not used yet, but we will in the future.
   license?: string;
```

Comment:
nit: WDYT about narrowing the type here?

```suggestion
  license?: 'PLATINUM';
```